### PR TITLE
New privacy policy 07.07.2022

### PR DESCRIPTION
**Pull request by halaprix**
# [Update the privacy policy](https://app.shortcut.com/oazo-apps/story/5242/update-the-privacy-policy)

  
## Changes 👷‍♀️
updated the privacy policy
  
## How to test 🧪
go to `/privacy` and read the entire text.
thank you

### CODE REVIEW

File: features/content/privacy/en.mdx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+The Site is using cookies. Cookies are small text files that are placed on your computer by websites that you visit. They are widely used in order to make websites work, or work more efficiently, as well as to provide information to the owners of the site. Cookies are typically stored on your computer's hard drive. Information collected from cookies is used by us to evaluate the effectiveness of our Site and analyze trends. The information collected from cookies allows us to determine such things as which parts of the Site are most visited and difficulties our visitors may experience in accessing the Site. With this knowledge, we can improve the quality of your experience on the Site by recognizing and delivering more of the most desired features and information, as well as by resolving access difficulties.
```
Comment (johnnieskywalker) [inline features/content/privacy/en.mdx:13]: line 25 (can't comment it as it wasn't edited). However not 100% sure about this one.
`Our service providers analyses this information and provides us with aggregate reports. `
shouldn't it be 
``Our service providers analyse this information and provide us with aggregate reports. ``

So one provider analyses but multiple providers analyse?

File: features/content/privacy/en.mdx [hunk 2]
```diff
@@ -36,7 +36,7 @@ Note that we are not responsible for your use of the Ethereum blockchain and you
 
 ### Your inquiries
 
-You may contact us by email to the following email address: legal@oasis.app. We use the data that you provide in an email to us, which you may give voluntarily, only in order to answer your contact question or to reply to your email in the best possible manner.
+You may contact us by email to the following email address: **[legal@oasis.app.](mailto:legal@oasis.app.)** We use the data that you provide in an email to us, which you may give voluntarily, only in order to answer your contact question or to reply to your email in the best possible manner.
```
Comment (johnnieskywalker) [inline features/content/privacy/en.mdx:39]: line 67
shouldn't it be 
`we no longer need` ? instead of `we no longer needs` ?

**Review by johnnieskywalker — CHANGES_REQUESTED** [review]

File: features/content/privacy/en.mdx [hunk 3]
```diff
@@ -1,16 +1,16 @@
 # Privacy Policy
 
-*Last updated: 1 June 2021*
+*Last updated: 7 June 2022*
 
-This Privacy Policy provides our privacy policy regarding the nature, purpose, use, and sharing of personal data or other information collected from the users of the website oasis.app and other websites which use subdomains of oasis.app (the "Site"). This Privacy Policy also includes the Cookie We are committed to protecting and respecting your privacy. Please read this carefully as this Privacy Policy is legally binding when you use the Site.
+This Privacy Policy provides our privacy policy regarding the nature, purpose, use, and sharing of personal data or other information collected from the users of the oasis.app website and other websites which use subdomains of oasis.app (the "Site"). . We are committed to protecting and respecting your privacy. Please read this carefully as this Privacy Policy is legally binding when you use the Site.
```
[the remark this answers is not in the document]
Comment (halaprix) [reply features/content/privacy/en.mdx]: copy paste from the docs ;| fixed

File: features/content/privacy/en.mdx (hunk 1 above)
Comment (halaprix) [reply features/content/privacy/en.mdx:13]: you are right - fixed

File: features/content/privacy/en.mdx (hunk 2 above)
Comment (halaprix) [reply features/content/privacy/en.mdx]: right - fixed. There were some other things grammarly caught.
But there are still things that make no sense to me, like `the processing is unlawful and you oppose the erasure of the personal data and requests instead the restriction of their use instead;`

**Pull request closed without merging**